### DISCUSSION TITLE
Unable to jog with keyboard?

### DISCUSSION THREAD

**Discussion by i-g-g-y** [winder/Universal-G-Code-Sender#1825] [Help]
Hello.

I must be doing something wrong. I can not jog with the keyboard. I set up my shortcuts and all, I click on the jog middle button with the keyboard picture and nothing happens.

Help?

**Comment by breiler** [comment]
What shortcuts have you assigned?

**Reply by i-g-g-y** [reply]
![image](https://user-images.githubusercontent.com/10361323/158637508-2336e497-bc54-483b-a6d0-6ec40490d378.png)

**Reply by i-g-g-y** [reply]
Rebooting my machine did the trick again. 

I can now jog with my keyboard. I can jog with another wireless keyboard, everything is cool now.

This is really weird.
